physical/raft/fsm.go

```diff
@@ -913,6 +922,22 @@ func (f *FSM) ApplyBatch(logs []*raft.Log) []interface{} {
 		panic("failed to store data")
 	}
 
+	if f.invalidateHook != nil {
+		go func() { // Call the callback asynchronously, to keep time we hold the db look low
```

Comment:
```suggestion
		go func() { // Call the callback asynchronously, to keep time we hold the db lock low
```

I do like this as well fwiw.